gradle.properties

```diff
@@ -1,21 +1,21 @@
 org.gradle.jvmargs=-Xmx2048M
 
-minecraft_version=1.20.4
+minecraft_version=1.20.6
 enabled_platforms=fabric,forge
 
 archives_base_name=brb
-mod_version=1.10.0+1.20.4
+mod_version=1.10.1+1.20.6
```

Comment:
Do we need to bump? I have never released 1.10.0 besides "rc"s

Reply:
That's fair enough. I had assumed it was the released version.